src/commands/suggestions/suggestionsUpdate.ts

```diff
@@ -0,0 +1,57 @@
+import { Message, MessageEmbed } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+
+import { AdminCommand } from '../../utils/commands';
+import {
+  suggestionStatesReadable,
+  getAvailableStatesString,
+  updateSuggestionState,
+  SuggestionState
+} from '../../components/suggestions';
+import { EMBED_COLOUR } from '../../utils/embeds';
+import { parseStateArg, validateUpdate } from './utils';
+
+class SuggestionsUpdateCommand extends AdminCommand {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'suggestions-update',
+      aliases: ['suggestions-update', 'suggestion-update'],
+      group: 'suggestions',
+      memberName: 'update',
+      args: [
+        {
+          key: 'value',
+          prompt: `enter one of ${getAvailableStatesString()} before your IDs.`,
```

Comment:
this will take in the entire string though, for example `actionable 1 2 3`, is it possible to have an arg just for the state, and another arg for the IDs string?